infrastructure/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "azurerm" {}
+
+module "service" {
+  source   = "git@github.com:contino/moj-module-webapp"
+  product  = "${var.product}-service"
+  location = "${var.location}"
+  env      = "${var.env}"
+  ilbIp    = "${var.ilbIp}"
+
+  app_settings = {
+    SERVICE_BUS_POLLING_DELAY_MS = "${var.service_bus_polling_delay_ms}"
```

Comment:
SERVICE_BUS_CONNECTION_STRING is not set here, as it contains a secret component (shared access key). It's supposed to be set manually in Azure until secret management is solved.